whois-rpsl/src/main/java/net/ripe/db/whois/common/ip/IpInterval.java

```diff
@@ -36,6 +36,7 @@ public static IpInterval<?> parse(final CIString prefix) {
 
     public static IpInterval<?> parse(final String prefix) {
         if (prefix.indexOf(':') == -1) {
+            // It could fail when prefix is taken from IpInterval.getKey().toString() for X.X.X.255 - X.X.X.0
```

Comment:
Remove unnecessary comment, it is too specific to this issue

Reply:
I will remove it but if in the future we use the tree together with this parse we will get an error again.
And the tree only accepts IpInterval
- tree.findFirstLessSpecific(IpInterval ipInterval)
- IpInterval.parse(Interval.toString) -> Could throw an error because of this issue